thirdparty/patchmatchnet/models/patchmatch.py

```diff
@@ -210,22 +209,26 @@ def forward(
         """Forward method for adaptive evaluation
 
         Args:
-            ref_feature: feature from reference view
-            src_features: features from source views
-            ref_proj: projection matrix of reference view
-            src_projs: source matrices of source views
-            depth_sample: sample depth map,
-            depth_min: minimum virtual depth,
-            depth_max: maximum virtual depth,
+            ref_feature: feature from reference view, (B, C, H, W)
+            src_features: features from (Nview-1) source views, (Nview-1) * (B, C, H, W), where Nview is the number of
+                input images (or views) of PatchmatchNet
+            ref_proj: projection matrix of reference view, (B, 4, 4)

```

Comment:
camera extrinsics? projection matrix would be a 3x4 matrix equal to K [ R | t ]

Reply:
the projection matrix defined by the author is 4x4 calculated by
```
proj_mat = extrinsics.copy()
proj_mat[:3, :4] = intrinsics @ proj_mat[:3, :4]
```